Anchor.toml

```diff
@@ -5,16 +5,16 @@ seeds = false
 skip-lint = false
 
 [programs.devnet]
-kyupad_smart_contract = "FHPbXuECjkYGrJHnwU2ozrm7TYzoo2c43CZTo3nJEzs"
+kyupad_smart_contract = "7xxCydutAXo4eicZXAN3JZmFCGg2dLqesGkrd8jAobxZ"
```

Comment:
We need to find a solution to put this on some environment configuration without making changes to any PR